.nvmrc

```
@@ -0,0 +1 @@
+12.22.12
```

How about just 12, so that it gets the latest version rather than a pinned one?
```suggestion
12
```